### DISCUSSION TITLE
Lockstep version tracks?

### DISCUSSION THREAD

**Discussion by fwendland** [InsertKoinIO/koin#1609] [Q&A]
We started to use Koin in one of our projects and had no problem to define one version for Koin across multiple packages (`koin-core` and `koin-test` to be precise) similar to #1389 [first comment](https://github.com/InsertKoinIO/koin/discussions/1389#discussioncomment-3412305). This seems to have changed with the latest release `3.4.2`. While `koin-core` has been released as 'v3.4.2', `koin-test` has remained as `v3.4.1`. My assumption of what packages have lockstep versions may have been wrong.

I've seen #1442 and understand that some libraries have received their own version track. I haven't seen `koin-test` among the milestones. Now I'm wondering:

- Does `koin-test` also have it's own version track (or is there a delay in `v3.4.2`)?
- Which Koin packages have lockstep versions?
- Are there version dependencies between `koin-test` and `koin-core` (or any other Koin packages) that one needs to consider when updating?
- Would it make sense to provide a BOM (possibly multiple) with compatible versions one can simply include in one's project?

Thanks for your help.

Best regards,
Flo

**Comment by arnaudgiuliani** [comment]
There have been some publication problem. All is aligned on 3.4.3 version now

**Reply by arnaudgiuliani** [reply]
A bom is also another solution yes

**Reply by fwendland** [reply]
Hi @arnaudgiuliani 

Thanks for the info. I've seen the alignment with the latest 3.4.3 release. In my case, I've split the version dependency into `core` and `test` and it works again.

Any solution to keep versions aligned and interoperable would help greatly in the future :)

**Reply by arnaudgiuliani** [reply]
Unfortunately, we had a CI/CD upload corruption. koin-core has been published and not koin-test. It was an exceptional case.

**Reply by arnaudgiuliani** [reply]
else it should koin-core & koin-test should be on the same track

**Comment by arnaudgiuliani** [comment]
Koin 3.5.0 is bringing BOM 🎉  You can use it with module `koin-bom` module
